src/main/java/ActivityFrame.java

```diff
@@ -95,7 +94,7 @@ public void setLink(String link) {
     }
 
     public void setActivity(String nextActivity) {
-        if (nextActivity == null)
+        if (nextActivity.contains("null"))
```

Comment:
Why would nextActivity contain null?

Reply:
When you request data from the API and there is no activity generated it returns null

Reply:
This is the kind of detail that needs a comment and a unit test, because this is pretty strange.

It returns a string containing "null"? 
Better if you compare it with .equal()

Reply:
Due to concatenation, the variable "nextActivity" says "null can be done with 0 people"
I added a comment.
The unit test is called noActivity()

Reply:
Not ideal. But I'll accept it.